fala/templates/base.html

```diff
@@ -89,7 +89,7 @@ <h1>
       ga('send', 'pageview');
     </script>
   {% endif %}
-  <script src="//maps.googleapis.com/maps/api/js"></script>
+  <script src="//maps.googleapis.com/maps/api/js?key={{ GOOGLE_MAPS_API_KEY }}"></script>
```

Comment:
Since this key is publicly visible, do we need to use secrets?

Do we have domain access control setup?

Reply:
We do.

And no, we don't need to use secrets. I went with them partly because they were set up initially, partly because it felt like a neat way of separating the config, and partly because I didn't really think about the fact that they are public.